README.md

```diff
@@ -151,6 +151,23 @@ script:
 
 The monitoring dashboard is accessible at `http://kubernetes-master-1.kubernetes-playground.local/monitoring/cluster`
 
+### Kites experiments
+
+Kites allows you to test the traffic exchanged between nodes
+and pods thanks to the use of the iperf3 and netsniff-ng tools.
```

Comment:
1) I'd remove everything after pods, so we don't have to keep that updated.
2) Use Nodes and Pods (as per Google style guide, when mentioning Kubernetes constructs)